packages/driver/src/cy/multi-domain/index.ts

```diff
@@ -155,6 +155,19 @@ export function addCommands (Commands, Cypress: Cypress.Cypress, cy: Cypress.cy,
           Cypress.backend('ready:for:domain', { originPolicy: location.originPolicy })
 
           if (err) {
+            if (err?.name === 'ReferenceError') {
```

Comment:
We limit adding the blurb about defining variables to just reference errors. This seems targeted enough to not warn on other random errors (like throwing errors, unsupported commands etc)